internal/util/archive.go

```diff
@@ -42,6 +42,85 @@ func CreateTarball(target string, contentPath string) error {
 	})
 }
 
+// UnpackTarball creates the target directory and unpacks the tarball into it.
```

Comment:
Not certain that this is worth it compared to just calling `tar -xzf` or similar which is likely to be handling more compression algorithms and other corner cases :)

Reply:
Good point, though omitting `-z` since I think that would force gzip. We can probably just make the target dir then do `tar -C <target> -xf <tarball>`